novawallet/Common/View/StackTable/StackTitleMultiValueEditCell.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import UIKit
+
+final class StackTitleMultiValueEditCell: RowView<GenericTitleValueView<IconDetailsView, GenericPairValueView<IconDetailsView, UILabel>>> {
```

Comment:
The problem with this implementation is that we must have some tappable UIControl in place of IconDetailsView. Probably, just user RoundedButton or UIButton